src/bindings/python/tests/test_graph/test_ops_binary.py

```diff
@@ -183,3 +183,44 @@ def test_power_v1():
     assert node.get_output_size() == 1
     assert list(node.get_output_shape(0)) == [8, 4, 6, 5]
     assert node.get_output_element_type(0) == Type.f32
+
+
+@pytest.mark.parametrize(
+    "graph_api_helper",
+    [ov.bitwise_and, ov.bitwise_or, ov.bitwise_xor],
+)
+@pytest.mark.parametrize(
+    "dtype",
+    [bool, np.int32],
+)
+def test_binary_bitwise_op(graph_api_helper, dtype):
+    shape = [2, 2]
```

Comment:
Can we use in this test different shapes to ensure the order of passing parameters and broadcast.

Reply:
Done